image/base/install-imagemagick

```diff
@@ -8,26 +8,24 @@ IMAGE_MAGICK_HASH="d282117bc6d0e91ad1ad685d096623b96ed8e229f911c891d83277b350ef8
 # We use debian, but GitHub CI is stuck on Ubuntu Bionic, so this must be compatible with both
```

Comment:
That is no longer true, is it? We could remove that logic and the comment?

Reply:
Indeed, we’re using our own images on GitHub CI, will remove :+1: